src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -175,26 +178,62 @@ export class SavedObjectsClient {
       return { saved_objects: [] };
     }
 
-    const docs = objects.map(doc => {
-      return { _type: get(doc, 'type'), _id: get(doc, 'id') };
-    });
+    const docs = objects.reduce((acc, { type, id }) => {
+      acc.push({}, createIdQuery({ type, id }));
+      return acc;
```

Comment:
It's the msearch header. We can leave it empty as we're specifying the index on the request instead of per query.